kubernetes-client/src/com/goyeau/kubernetes/client/KubeConfig.scala

```diff
@@ -37,4 +32,45 @@ object KubeConfig {
     fromFile(kubeconfig, contextName)
   def fromFile[F[_]: Sync: Logger](kubeconfig: File, contextName: String): F[KubeConfig] =
     Yamls.fromKubeConfigFile(kubeconfig, Option(contextName))
+
+  def of[F[_]: ApplicativeError[*[_], Throwable]](
```

Comment:
With regards to Scala 3 that comes with wildcard argument in types, will it be built via kind-projector?
It seems k8s-client for Scala3 would not even need kind-projector dependency.